### ISSUE TITLE
【EESS】New submissions for Tue, 31 Oct 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#822]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### Promise:Prompt-driven 3D Medical Image Segmentation Using Pretrained  Image Foundation Models
 - **Authors:** Hao Li, Han Liu, Dewei Hu, Jiacheng Wang, Ipek Oguz
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2310.19721
 - **Pdf link:** https://arxiv.org/pdf/2310.19721
 - **Abstract**
 To address prevalent issues in medical imaging, such as data acquisition challenges and label availability, transfer learning from natural to medical image domains serves as a viable strategy to produce reliable segmentation results. However, several existing barriers between domains need to be broken down, including addressing contrast discrepancies, managing anatomical variability, and adapting 2D pretrained models for 3D segmentation tasks. In this paper, we propose ProMISe,a prompt-driven 3D medical image segmentation model using only a single point prompt to leverage knowledge from a pretrained 2D image foundation model. In particular, we use the pretrained vision transformer from the Segment Anything Model (SAM) and integrate lightweight adapters to extract depth-related (3D) spatial context without updating the pretrained weights. For robust results, a hybrid network with complementary encoders is designed, and a boundary-aware loss is proposed to achieve precise boundaries. We evaluate our model on two public datasets for colon and pancreas tumor segmentations, respectively. Compared to the state-of-the-art segmentation methods with and without prompt engineering, our proposed method achieves superior performance. The code is publicly available at https://github.com/MedICL-VU/ProMISe.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
### Improved Lossless Coding for Storage and Transmission of Multichannel  Immersive Audio
 - **Authors:** Toni Hirvonen, Mahmoud Namazi
 - **Subjects:** Audio and Speech Processing (eess.AS); Multimedia (cs.MM)
 - **Arxiv link:** https://arxiv.org/abs/2310.18461
 - **Pdf link:** https://arxiv.org/pdf/2310.18461
 - **Abstract**
 In this paper, techniques for improving multichannel lossless coding are examined. A method is proposed for the simultaneous coding of two or more different renderings (mixes) of the same content. The signal model uses both past samples of the upmix, and the current time samples of downmix samples to predict the upmix. Model parameters are optimized via a general linear solver, and the prediction residual is Rice coded. Additionally, the use of an SVD projection prior to residual coding is proposed. A comparison is made against various baselines, including FLAC. The proposed methods show improved compression ratios for the storage and transmission of immersive audio.
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
### Transport-of-Intensity Model for Single-Mask X-ray Differential Phase  Contrast Imaging
 - **Authors:** Jingcheng Yuan, Mini Das
 - **Subjects:** Image and Video Processing (eess.IV); Instrumentation and Detectors (physics.ins-det); Medical Physics (physics.med-ph); Optics (physics.optics); Tissues and Organs (q-bio.TO)
 - **Arxiv link:** https://arxiv.org/abs/2310.19087
 - **Pdf link:** https://arxiv.org/pdf/2310.19087
 - **Abstract**
 X-ray phase contrast imaging has emerged as a promising technique for enhancing contrast and visibility of light-element materials, including soft tissues and tumors. In this paper, we propose a novel model for a single-mask phase imaging system based on the transport-of-intensity equation. Our model offers an intuitive understanding of signal and contrast formation in single-mask phase imaging systems. We also demonstrate efficient retrieval of attenuation and differential phase contrast with just one intensity image without requiring spectral information or mask/detector movement. The model validity as well as the proposed retrieval method is demonstrated via both experimental results on a system developed in-house as well as with Monte Carlo simulations. Our proposed model overcomes the limitations of existing models by providing an intuitive visualization of the image formation process. It also allows optimizing differential phase imaging geometries for practical applications, further enhancing broader applicability. Furthermore, the general methodology described herein offers insight on deriving transport-of-intensity models for novel X-ray imaging systems with periodic structures in the beam path.
## Keyword: video retrieval
There is no result 
## Keyword: mobile
There is no result 
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Improved Lossless Coding for Storage and Transmission of Multichannel  Immersive Audio
 - **Authors:** Toni Hirvonen, Mahmoud Namazi
 - **Subjects:** Audio and Speech Processing (eess.AS); Multimedia (cs.MM)
 - **Arxiv link:** https://arxiv.org/abs/2310.18461
 - **Pdf link:** https://arxiv.org/pdf/2310.18461
 - **Abstract**
 In this paper, techniques for improving multichannel lossless coding are examined. A method is proposed for the simultaneous coding of two or more different renderings (mixes) of the same content. The signal model uses both past samples of the upmix, and the current time samples of downmix samples to predict the upmix. Model parameters are optimized via a general linear solver, and the prediction residual is Rice coded. Additionally, the use of an SVD projection prior to residual coding is proposed. A comparison is made against various baselines, including FLAC. The proposed methods show improved compression ratios for the storage and transmission of immersive audio.
### Designing a Kinetic Façade Using BB-BC Algorithm with a Focus on  Enhancing Building Energy Efficiency
 - **Authors:** Marzieh Soltani, Arash Atashi
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2310.18650
 - **Pdf link:** https://arxiv.org/pdf/2310.18650
 - **Abstract**
 In order to increase energy efficiency in buildings, optimizing the parameters of the facade form can be challenging due to the dynamic nature of solar radiation. One effective solution is the use of kinetic facades as a second skin, which can control energy consumption. This study proposes a parametric kinetic facade to increase building energy efficiency, along with a framework to optimize its form using the Bang-Big Crunch (BB-BC) optimization algorithm. The study involved modeling a two-story office building in Shiraz city and calculating the energy consumption resulting from building operation over a three-day period without considering the second skin of the facade. In the second stage of the study, the second skin was optimized for the same three-day interval and calculated as a parametric, static facade. In the last step, the parameters of the second skin were optimized for three one-day intervals, assuming the possibility of kinematic changes each day. The total energy consumption of building operation for the three days was then calculated and analyzed.The Python programming language was used to develop the optimization algorithm, while Rhino software, and Grasshopper, Ladybug, and Honeybee plugins were used for building modeling and simulation of light, energy, and weather parameters.The results of the study demonstrate the effectiveness of the proposed kinetic facade and the proper performance of the proposed algorithm for solving similar problems. The study found that the use of the second skin with kinetic function reduced energy consumption by 28%. Additionally, the results from the second and third stages of the study showed that the use of the second facade shell with kinematic function, compared to its function in static mode, reduced energy consumption by 4%.Overall...
### Subjective Quality Evaluation of Point Clouds Using a Head Mounted  Display
 - **Authors:** Joao Prazeres, Rafael Rodrigues, Manuela Pereira, Antonio M. G. Pinheiro
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2310.19179
 - **Pdf link:** https://arxiv.org/pdf/2310.19179
 - **Abstract**
 This paper reports on a subjective quality evaluation of static point clouds encoded with the MPEG codecs V-PCC and G-PCC, the deep learning-based codec RS-DLPCC, and the popular Draco codec. 18 subjects visualized 3D representations of distorted point clouds using a Head Mounted Display, which allowed for a direct comparison with their reference. The Mean Opinion Scores (MOS) obtained in this subjective evaluation were compared with the MOS from two previous studies, where the same content was visualized either on a 2D display or a 3D stereoscopic display, through the Pearson Correlation, Spearman Rank Order Correlation, Root Mean Square Error, and the Outlier Ratio. The results indicate that the three studies are highly correlated with one another. Moreover, a statistical analysis between all evaluations showed no significant differences between them.
### Data-driven optimal control via linear programming: boundedness  guarantees
 - **Authors:** Lucia Falconi, Andrea Martinelli, John Lygeros
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2310.19563
 - **Pdf link:** https://arxiv.org/pdf/2310.19563
 - **Abstract**
 The linear programming (LP) approach is, together with value iteration and policy iteration, one of the three fundamental methods to solve optimal control problems in a dynamic programming setting. Despite its simple formulation, versatility, and predisposition to be employed in model-free settings, the LP approach has not enjoyed the same popularity as the other methods. The reason is the often poor scalability of the exact LP approach and the difficulty to obtain bounded solutions for a reasonable amount of constraints. We mitigate these issues here, by investigating fundamental geometric features of the LP and developing sufficient conditions to guarantee finite solutions with minimal constraints. In the model-free context, we show that boundedness can be guaranteed by a suitable choice of dataset and objective function.
### Seeing Through the Conversation: Audio-Visual Speech Separation based on  Diffusion Model
 - **Authors:** Suyeon Lee, Chaeyoung Jung, Youngjoon Jang, Jaehun Kim, Joon Son Chung
 - **Subjects:** Audio and Speech Processing (eess.AS); Computer Vision and Pattern Recognition (cs.CV); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2310.19581
 - **Pdf link:** https://arxiv.org/pdf/2310.19581
 - **Abstract**
 The objective of this work is to extract target speaker's voice from a mixture of voices using visual cues. Existing works on audio-visual speech separation have demonstrated their performance with promising intelligibility, but maintaining naturalness remains a challenge. To address this issue, we propose AVDiffuSS, an audio-visual speech separation model based on a diffusion mechanism known for its capability in generating natural samples. For an effective fusion of the two modalities for diffusion, we also propose a cross-attention-based feature fusion mechanism. This mechanism is specifically tailored for the speech domain to integrate the phonetic information from audio-visual correspondence in speech generation. In this way, the fusion process maintains the high temporal resolution of the features, without excessive computational requirements. We demonstrate that the proposed framework achieves state-of-the-art results on two benchmarks, including VoxCeleb2 and LRS3, producing speech with notably better naturalness.
### Density Estimation for Entry Guidance Problems using Deep Learning
 - **Authors:** Jens A. Rataczak, Davide Amato, Jay W. McMahon
 - **Subjects:** Systems and Control (eess.SY); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2310.19684
 - **Pdf link:** https://arxiv.org/pdf/2310.19684
 - **Abstract**
 This work presents a deep-learning approach to estimate atmospheric density profiles for use in planetary entry guidance problems. A long short-term memory (LSTM) neural network is trained to learn the mapping between measurements available onboard an entry vehicle and the density profile through which it is flying. Measurements include the spherical state representation, Cartesian sensed acceleration components, and a surface-pressure measurement. Training data for the network is initially generated by performing a Monte Carlo analysis of an entry mission at Mars using the fully numerical predictor-corrector guidance (FNPEG) algorithm that utilizes an exponential density model, while the truth density profiles are sampled from MarsGRAM. A curriculum learning procedure is developed to refine the LSTM network's predictions for integration within the FNPEG algorithm. The trained LSTM is capable of both predicting the density profile through which the vehicle will fly and reconstructing the density profile through which it has already flown. The performance of the FNPEG algorithm is assessed for three different density estimation techniques: an exponential model, an exponential model augmented with a first-order fading-memory filter, and the LSTM network. Results demonstrate that using the LSTM model results in superior terminal accuracy compared to the other two techniques when considering both noisy and noiseless measurements.
### Optimizing Logical Execution Time Model for Both Determinism and Low  Latency
 - **Authors:** Sen Wang, Dong Li, Ashrarul H. Sifat, Shao-Yu Huang, Xuanliang Deng, Changhee Jung, Ryan Williams, Haibo Zeng
 - **Subjects:** Systems and Control (eess.SY); Operating Systems (cs.OS); Symbolic Computation (cs.SC)
 - **Arxiv link:** https://arxiv.org/abs/2310.19699
 - **Pdf link:** https://arxiv.org/pdf/2310.19699
 - **Abstract**
 The Logical Execution Time (LET) programming model has recently received considerable attention, particularly because of its timing and dataflow determinism. In LET, task computation appears always to take the same amount of time (called the task's LET interval), and the task reads (resp. writes) at the beginning (resp. end) of the interval. Compared to other communication mechanisms, such as implicit communication and Dynamic Buffer Protocol (DBP), LET performs worse on many metrics, such as end-to-end latency (including reaction time and data age) and time disparity jitter. Compared with the default LET setting, the flexible LET (fLET) model shrinks the LET interval while still guaranteeing schedulability by introducing the virtual offset to defer the read operation and using the virtual deadline to move up the write operation. Therefore, fLET has the potential to significantly improve the end-to-end timing performance while keeping the benefits of deterministic behavior on timing and dataflow. To fully realize the potential of fLET, we consider the problem of optimizing the assignments of its virtual offsets and deadlines. We propose new abstractions to describe the task communication pattern and new optimization algorithms to explore the solution space efficiently. The algorithms leverage the linearizability of communication patterns and utilize symbolic operations to achieve efficient optimization while providing a theoretical guarantee. The framework supports optimizing multiple performance metrics and guarantees bounded suboptimality when optimizing end-to-end latency. Experimental results show that our optimization algorithms improve upon the default LET and its existing extensions and significantly outperform implicit communication and DBP in terms of various metrics, such as end-to-end latency, time disparity, and its jitter.
### Promise:Prompt-driven 3D Medical Image Segmentation Using Pretrained  Image Foundation Models
 - **Authors:** Hao Li, Han Liu, Dewei Hu, Jiacheng Wang, Ipek Oguz
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2310.19721
 - **Pdf link:** https://arxiv.org/pdf/2310.19721
 - **Abstract**
 To address prevalent issues in medical imaging, such as data acquisition challenges and label availability, transfer learning from natural to medical image domains serves as a viable strategy to produce reliable segmentation results. However, several existing barriers between domains need to be broken down, including addressing contrast discrepancies, managing anatomical variability, and adapting 2D pretrained models for 3D segmentation tasks. In this paper, we propose ProMISe,a prompt-driven 3D medical image segmentation model using only a single point prompt to leverage knowledge from a pretrained 2D image foundation model. In particular, we use the pretrained vision transformer from the Segment Anything Model (SAM) and integrate lightweight adapters to extract depth-related (3D) spatial context without updating the pretrained weights. For robust results, a hybrid network with complementary encoders is designed, and a boundary-aware loss is proposed to achieve precise boundaries. We evaluate our model on two public datasets for colon and pancreas tumor segmentations, respectively. Compared to the state-of-the-art segmentation methods with and without prompt engineering, our proposed method achieves superior performance. The code is publicly available at https://github.com/MedICL-VU/ProMISe.
### Transmission line condition prediction based on semi-supervised learning
 - **Authors:** Sizhe Li, Xun Ma, Nan Liu, Yi Jin
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2310.19756
 - **Pdf link:** https://arxiv.org/pdf/2310.19756
 - **Abstract**
 Transmission line state assessment and prediction are of great significance for the rational formulation of operation and maintenance strategy and improvement of operation and maintenance level. Aiming at the problem that existing models cannot take into account the robustness and data demand, this paper proposes a state prediction method based on semi-supervised learning. Firstly, for the expanded feature vector, the regular matrix is used to fill in the missing data, and the sparse coding problem is solved by representation learning. Then, with the help of a small number of labelled samples to initially determine the category centers of line segments in different defective states. Finally, the estimated parameters of the model are corrected using unlabeled samples. Example analysis shows that this method can improve the recognition accuracy and use data more efficiently than the existing models.
